### ISSUE TITLE
Configuring Project ID is a bit confusing

### ISSUE THREAD

**Issue by bshaffer** [GoogleCloudPlatform/cloud-trace-nodejs#675]
I received the following error even though `GOOGLE_PROJECT_ID` was set as an environment variable. I assume it’s because a project NUMBER is requested, but the error message does not explain which environment variable should be set.

```
ERROR:@google-cloud/trace-agent: Unable to acquire the project number from metadata service. Please provide a valid project number as an env. variable, or through config.projectId passed to start(). Error: getaddrinfo ENOTFOUND metadata.google.internal metadata.google.internal:80
```

So there are two issues here:

1. The `GOOGLE_PROJECT_ID` environment variable should work to configure the project ID in the config
2. The error message should display the name of the env variable

Additionally, is the explanation "through config.projectId passed to start()" clear enough for explaining the following:

```nodejs
start({ projectId: "YOUR_PROJECT_ID"})
```

Because I found this to be a little unclear.

**Comment by kjin** [comment]
We support `GCLOUD_PROJECT` being read from the environment. I can't find any Google-backed documentation for the usage of `GOOGLE_PROJECT_ID` -- if you have a source please let us know.

It's true that the error message is somewhat outdated (and a little unclear) -- thanks for bringing this to our attention!

**Comment by bshaffer** [comment]
Thank you for your quic response, `GCLOUD_PROJECT` works great! 

So then my only ask is to make the error message use "project ID" instead of "project number" and identify `GCLOUD_PROJECT` as the env var.

**Comment by kjin** [comment]
Fixed in #700
